docs/Esport Protocol V1/Tech integration/api-touchpoints.md

```diff
@@ -256,59 +265,55 @@ The response is a JSON object containing various details about the wallet's bala
-- `lazy_balance`: The current balance in the wallet that hasn't been committed to the chain.
-- `on_chain_balance`: The balance that is recorded on the blockchain.
+- `created_at`: Timestamp for when the wallet record was created.
+- `modified_at`: Timestamp for the last modification to the wallet record.
+- `minWalletBalToWithdraw`: The minimum balance amount that is required to withdraw. 
 
 #### Example Response
 ```json
 {
     "wallet_address": "[WALLET_ADDRESS]",
     "earned_balance": 0,
     "bought_balance": 0,
-    "bonus_balance": 0,
+    "bonus_balance": 50,
+    "lazy_balance": 50,
+    "on_chain_balance": 0,
     "wallet_transaction_error": false,
-    "created_at": "2024-01-29T17:53:26.934052",
-    "modified_at": "2024-01-29T17:53:26.934052",
-    "lazy_balance_delta": 0,
-    "is_lazy_balance_updated": true,
-    "lazy_balance": 0,
-    "on_chain_balance": 0
+    "lazy_balance_delta": 50,
+    "is_lazy_balance_updated": false,
+    "created_at": "",
+    "modified_at": "",
+    "minWalletBalToWithdraw": 0
 }
-
-
 ```
 
 # API Documentation: Wallet Transaction History API
 
-## Endpoint: GET `/api/v1/wallet/lazyChain/transactions/:wallet_address?token={token}`
-This endpoint retrieves the transaction history of a specific wallet address, providing a detailed view of all transactions associated with it.
+## Endpoint: GET `/fe/wallet/transactions`
+This endpoint retrieves the transaction history of a specific user, providing a detailed view of all transactions associated with it.
 
 #### Request Header
 - `Content-Type`: application/json
-- `x-clientID`: [Your Client ID] – Your unique client identifier.
-- `x-clientSecret`: [Your Client Secret] – Your client secret for authentication.
-
-### Path Parameters
-- `wallet_address`: The unique blockchain address of the wallet whose transaction history is being requested.
+- `user-jwt-token`: Users JWT token for authentication.
 
 ### Response Structure
-The response is a JSON object with two key arrays: `fromTransactions` and `toTransactions`. Each array contains transaction objects detailing the history of transactions.
+The response is a JSON object with `last_evaluated_key` for pagination and three key arrays: `fromTransactions`, `toTransactions` and `transactions`. Each array contains transaction objects detailing the history of transactions.
 
-- `fromTransactions`: Transactions sent from the specified wallet.
-- `toTransactions`: Transactions received by the specified wallet.
+- `fromTransactions`: Transactions sent from the specified user.
+- `toTransactions`: Transactions received by the specified user.
+- `transactions`: Transactions by the specified user.
 
 Each transaction object includes fields such as:
 - `uid`: Unique identifier of the transaction.
 - `from_wallet`: Sender's masked wallet address.
 - `to_wallet`: Receiver's masked wallet address.
 - `amount`: Amount of the transaction.
```

Comment:
@Jitendra-08 Can we add the key
`"description": {
                "platformProtocolsTxnAmount": 50,
                "platformProtocolsTxnDescription": "Tournament fee deduct",
                "platformProtocolsTxnName": "Tournament fee deduct",
                "type": "JOIN"
            },`
            
            and mention about this. So that it will easy for game team to understand the description that they have to use in FE